spec/fixtures/packages/example/lib/example.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  activate () {}
+}
```

Comment:
I think that in order to activate, this package might need a `package.json`. Alternatively, you could probably use one of the existing packages in the `fixtures/packages` folder for this purpose.

Reply:
Thanks for the suggestion, @maxbrunsfeld. It seems to have been an issue with the package directory path. a802ba3cc3ced27d66532db04b4f7d28b163774d resolves the issue. I'm not sure why I didn't need to set that path for the tests to pass on my local dev machine, but at least now the tests pass locally and on CI. 😅